.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - '8'
```

Comment:
It's been kind of annoying to run 2 sets of the test suite in travis. I can move this to another PR if you care.